internal/entities/command_deployment.go

```diff
@@ -127,3 +142,23 @@ func parseCustomAttributes(a *[]string) (*map[string]string, error) {
 
 	return &customAttributeMap, nil
 }
+
+func getCustomAttributes(a []string, b string) (interface{}, error) {
```

Comment:
Can we make this function always return `*map[string]interface{}` or `map[string]interface{}` instead of `interface{}`? Then you won't need to use the [switch](https://github.com/newrelic/newrelic-cli/pull/1526/files#diff-dddd8b36a52df433365ab87762f713ed32cb9b0282b5a2972c63608c5394bb4bR75-R80) statement.

Reply:
:butting-in-emoji: `map[string]interface{}`'s zero value is nil since it's a reference type, so no need for`*map[string]interface{}`

Reply:
Thanks, done in ed2172c698e2995c283c80c7c47d106cb1fc9f45